install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  ack-4.8-kube-1.22-api-removals-in-4.9: |
+      Kubernetes 1.22 and thus OpenShift 4.9 remove several APIs which require admin consideration. Please see
```

```suggestion
      Kubernetes 1.22 and therefore OpenShift 4.9 remove several APIs which require admin consideration. Please see
```

i dunno, "thus" stuck out to me, but maybe it's just me.

---

therefore, I changed it :)

---

thus, it looks good to me.

wait, what? :)